go/obscuronode/enclave/state.go

```diff
@@ -228,17 +245,18 @@ func findRoundWinner(receivedRollups []*Rollup, parent *Rollup, parentState Stat
 
 // mutates the state
 // process deposits from the proof of the parent rollup(exclusive) to the proof of the current rollup
-func processDeposits(fromBlock *types.Block, toBlock *types.Block, s RollupState, blockResolver BlockResolver) RollupState {
+func processDeposits(fromBlock *types.Block, toBlock *types.Block, blockResolver BlockResolver) []nodecommon.L2Tx {
```

Comment:
Can we update the comments ? Don't think they're correct anymore

Reply:
good point